test-case/check-sof-logger.sh

```diff
@@ -47,23 +47,34 @@ if [[ ! -f $ldcFile ]]; then
 fi
 dlogi "Found file: $(md5sum $ldcFile|awk '{print $2, $1;}')"
 
-tmp_file=/tmp/$RANDOM.sof-logger.log
+rdnum=$RANDOM
+tmp_file=/tmp/$rdnum.logger.log
+err_tmp_file=/tmp/$rdnum.err_logger.log
 
 func_lib_check_sudo
 
 dlogi "Try to dump the dma trace log via sof-logger ..."
-dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &"
-sudo $loggerBin -t -l $ldcFile -o $tmp_file 2>&1 &
+# sof-logger errors will output to $err_tmp_file
+dlogc "sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &"
+sudo $loggerBin -t -l $ldcFile -o $tmp_file 2> $err_tmp_file &
 sleep 2
 dlogc "sudo pkill -9 $(basename $loggerBin)"
 sudo pkill -9 $(basename $loggerBin) 2> /dev/null
 
+# check if we get any sof-logger errors
+logger_err=`grep -i "error" $err_tmp_file`
```

Comment:
if error record at `$tmp_file` ??

here 2 just for the stderr, but if 'error' dump into stdout

`logger_err=`grep -i "error"  $tmp_file $err_tmp_file``

Reply:
@Bin-QA @aiChaoSONG The logger update is to output error message to both stderr and stdout. So we need to only check stderr to check if error happen.

Reply:
@Bin-QA @xiulipan if FW got error, the will be "error" keyword in the log,  checking both file, will break the logic.

Reply:
if in both, why need `$err_tmp_file` if not in both, miss `$tmp_file` whether will lost something?